Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Paged/WalletSolanaProviderScript.js

```diff
@@ -35,7 +35,7 @@ window.__firefox__.execute(function($, $Object, $Function, $Array) {
           "securityToken": SECURITY_TOKEN,
           "method": method,
           "args": JSON.stringify(payload, (key, value) => {
-            /* JSON.stringify will convert Uint8Array to a dictionary, f we convert to an array we get a list. */
+            /* JSON.stringify will convert Uint8Array to a dictionary, if we convert to an array we get a list. */
             return value instanceof Uint8Array ? Array.from(value) : value;
```

Comment:
Just pointing out that both `Promise ` and `Array.from` are also overridable. The `Promise` looked concerning, but since we are using `$()` wrapper it's not possible (afaik) to get the body by overriding `Promise`. `Array.from` is probably fine to be kept as is as well

Reply:
> but since we are using $() wrapper it's not possible (afaik) to get the body by overriding Promise. Array.from is probably fine to be kept as is as well

Spoke with @Brandon-T and he agreed with this too. I can hold merge until @kdenhartog has a chance to look (if needed?), but I think we're alright leaving as-is for now 👍.

Reply:
I can't think of any particular concerns with this currently either